Content.Shared/Damage/Components/PassiveDamageComponent.cs

```diff
@@ -12,29 +13,52 @@ namespace Content.Shared.Damage.Components;
 public sealed partial class PassiveDamageComponent : Component
```

Comment:
Should probably also have `[Access(typeof(PassiveDamageSystem))]` and `[AutoGenerateComponentState]` with `[AutoNetworkedField]` for the properties.